packages/e2e/features/ui/components/storage/storage-browser/download.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Download on Storage Browser
+
+  Background:
+    Given I'm running the example "ui/components/storage/storage-browser/default-auth"
+
+  @react
+  Scenario: Download is available for files
+    When I type my "email" with status "CONFIRMED"
+    Then I type my password
+    Then I click the "Sign in" button
+    When I click the first button containing "public"
+    Then I see download for file "001_dont_delete_file.txt"
```

Comment:
nit:
Should we move this file inside `DO_NOT_DELETE` folder. That way all required items for e2e are in one place

Reply:
+1 to this. Should do a follow-up to reorganize data for testing